packages/core/streams/openai-stream.test.tsx

```diff
@@ -104,41 +125,27 @@ describe('OpenAIStream', () => {
   it('should handle backpressure on the server', async () => {
     const controller = new AbortController();
     const stream = OpenAIStream(
-      await fetch(server.api, {
-        headers: {
-          'x-mock-service': 'openai',
-          'x-mock-type': 'chat',
-        },
+      await fetch(DEFAULT_TEST_URL, {
         signal: controller.signal,
       }),
     );
     const response = new StreamingTextResponse(stream);
     const client = createClient(response);
     const chunks = await client.readAndAbort(controller);
+
     expect(JSON.stringify(chunks)).toMatchInlineSnapshot(`"[\\"Hello\\"]"`);
-    expect(JSON.stringify(server.getRecentFlushed())).toMatchInlineSnapshot(
-      `"[{\\"id\\":\\"chatcmpl-7RyNSW2BXkOQQh7NlBc65j5kX8AjC\\",\\"object\\":\\"chat.completion.chunk\\",\\"created\\":1686901302,\\"model\\":\\"gpt-3.5-turbo-0301\\",\\"choices\\":[{\\"delta\\":{\\"role\\":\\"assistant\\"},\\"index\\":0,\\"finish_reason\\":null}]},{\\"id\\":\\"chatcmpl-7RyNSW2BXkOQQh7NlBc65j5kX8AjC\\",\\"object\\":\\"chat.completion.chunk\\",\\"created\\":1686901302,\\"model\\":\\"gpt-3.5-turbo-0301\\",\\"choices\\":[{\\"delta\\":{\\"content\\":\\"Hello\\"},\\"index\\":0,\\"finish_reason\\":null}]}]"`,
```

Comment:
How important is the flush testing here? Does the first expect cover all that's needed or does this change remove coverage?